agent/06_agent_create_cluster.sh

```diff
@@ -152,6 +162,40 @@ function attach_appliance_diskimage() {
     done
 }
 
+function attach_agent_iso_no_registry() {
+    set_file_acl
+
+    local base_dir=$SCRIPTDIR/$OCP_DIR
+    local iso_name="agent-ove.$(uname -p).iso"
+    local agent_iso_no_registry=$(find "$base_dir" -type f -name "$iso_name" 2>/dev/null | head -n 1)
+
+    for (( n=0; n<${2}; n++ ))
+    do
+        name=${CLUSTER_NAME}_${1}_${n}
+        sudo virt-xml ${name} --add-device --disk "${agent_iso_no_registry}",device=cdrom,target.dev=sdc
+        sudo virt-xml ${name} --edit target=sda --disk="boot_order=1"
+        sudo virt-xml ${name} --edit target=sdc --disk="boot_order=2" --start
+        echo "Waiting for 3 mins to arrive at agent-tui screen"
+        sleep 180
```

Comment:
I don't think it's necessary to wait 3 minutes for each node, it should be quicker to start all of them and _then_ wait for no more than 3 minutes in total

Reply:
If we don't wait for the agent TUI to show up and finish the automated steps, each VM just boots up and all of them waits at the first screen of TUI to enter rendezvousIP and then code execution just proceeds to the point where it expects the assisted install UI to be up but in reality, code execution never gets a chance to set rendezvousIP.

Reply:
There's no need to serialize them, and to wait NUM_MASTERS * NUM_WORKERS * 3 minutes (in the compact case 9 additional minutes). So:

* Start all the instances simultaneously
* Wait 3 minutes
* Loop over all the instances and set the agent TUI accordingly on the role